base/applications/taskmgr/procpage.c

```diff
@@ -931,3 +935,202 @@ int CALLBACK ProcessPageCompareFunc(LPARAM lParam1, LPARAM lParam2, LPARAM lPara
     }
     return ret;
 }
+
+static BOOL NormalizeDevicePath(LPWSTR lpPath, DWORD dwLength)
+{
+    WCHAR *pszPath;
+    BOOL bSuccess = FALSE;
+
+    /* Check if lpPath is a device path */
+    if (_wcsnicmp(lpPath, L"\\Device\\", 8) != 0)
+    {
+        return FALSE;
+    }
+
+    pszPath = HeapAlloc(GetProcessHeap(), 0, dwLength * sizeof(WCHAR));
+
+    if (!pszPath)
+    {
+        return FALSE;
+    }
+
+    for (WCHAR cDrive = L'A'; cDrive <= L'Z'; cDrive++)
```

Comment:
Please move the declaration WCHAR cDrive out of the for-loop to the top of the function to make this code compatible to more compilers.

Reply:
OK.

Reply:
All those compilers are unsupported, and Joachim does not send PR's himself so feel free to ignore this request.